tic_tac_toe/game.py

```diff
@@ -63,42 +81,129 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
-
-
+    # pdb.set_trace()
+    _in_a_row_to_win = len(board[0])
+    vertical_combos = []
+    diagonal_combos_1 = []
+    diagonal_combos_2 = []
+    list_index = 0
+    flat_win_check = []
+    flat_win_check += board
+
+    for i in range(_in_a_row_to_win):
+        vert_list = []
+        for _list in board:
+            vert_list.append(_list[i])
+        vertical_combos.append(vert_list)
+    flat_win_check += vertical_combos
+    
+    for i in range(_in_a_row_to_win):
+        diagonal_combos_1 += [board[i][i]]
+    #return diagonal_combos_1
+    flat_win_check += [diagonal_combos_1]
+    
+    for i in reversed(range(_in_a_row_to_win)):
+        diagonal_combos_2 += [board[list_index][i]]
+        list_index += 1
+    #return diagonal_combos_2
+    flat_win_check += [diagonal_combos_2]
+    #return flat_win_check
+
+    for _list in flat_win_check:
+        if _list[0] == player and len(set(_list)) == 1:
+            return player
+    return None
+                
```

Comment:
I see that you've written this to handle any size of board, however you're not using your `_is_winning_combination` helper function. This is intended to be written in a way that you're passing the board, player, and a set of coordinates to the helper function to test if it's a winning combination. Otherwise the helper function is useless.